src/App.js

```diff
@@ -1,23 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { searchRecipes, getRecipe } from "./api";
+import RecipeSearch from "./component/RecipeSearch";
+import RecipeList from "./component/RecipeList";
+import DisplayRecipe from "./component/DisplayRecipe";
 
 function App() {
+  const [recipes, setRecipes] = useState([]);
+  const handleSubmit = async (term) => {
+    const results = await searchRecipes(term);
+    setRecipes(results.recipes);
+  };
+
+  const [recipe, setRecipe] = useState([]);
```

Comment:
Why is useState's default value an empty array?

Reply:
Why can't it be an empty array? Is it because it is supposed to receive an object later?
Btw, in case the recipe's default value is an empty array, why doesn't this conditional rendering `{recipe && <Recipe recipe={recipe} />}` throw any error even when an empty array should be truthy?